protocol_rfcs/index.md

```diff
@@ -1,5 +1,23 @@
 ## Table of RFCs
 
-Format: <RFC file name>, proposed <date>, accepted/rejected <date> 
+This file contains the list of all the protocol RFCs that have been proposed. The details of each RFC are present in the associated .md file. 
+
+# Proposed
+
+| RFC name | RFC file name | Github issue | Date proposed |
+|-|-|-|-|
+| My Table Feature | my_table_feature.md | https://github.com/delta-io/delta/issues/XXXX | 2023-02-05 | <!-- remove this when adding the first RFC -->
```

Comment:
This new proposal seems to be the best candidate to add as an example - https://github.com/delta-io/delta/issues/2598 

How about we work together to add that using this process? it will be a good real exercise to polish this process.

Reply:
ohh i agree. lets use this! 
thank you for your cooperation!